using/legal/data-protection-policy.md

```diff
@@ -0,0 +1,33 @@
+# Data Protection Policy
+
+## Overview
+
+The GDPR (General Data Protection Regulation) is UK legislation that governs how we process data originating from users from the UK.
+Exercism aims to be GDPR compliant for all users, even those outside the EU and you therefore have some important rights applicable to our Platform.
```

Comment:
Isn't GDPR used across the EU: https://en.wikipedia.org/wiki/General_Data_Protection_Regulation?

Reply:
We're not part of the EU any more, so we just copied all the laws and are now governed by the UK ones. GDPR isn't actually a law though, even in the EU. It's a framework that each country interprets and creates their own law on.

Reply:
Okay. Thanks for explaining

Reply:
> We're not part of the EU any more, so we just copied all the laws and are now governed by the UK ones. GDPR isn't actually a law though, even in the EU. It's a framework that each country interprets and creates their own law on.

This is not true. _Regulations_ are laws that immediately become enforceable in all member states. _Directives_ are more like the frameworks that need to be interpreted and turned into national law.